SPECS/kexec-tools/51_kexec_tools.cfg

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Do we actually need these headers? (just curious).

Reply:
It would feel weird to not include a shebang. I did look into it out of curiosity, and it seems like scripts sourced from other bash scripts don't need a shebang. 

https://stackoverflow.com/questions/7615430/do-you-need-shebang-in-all-bash-scripts

Reply:
I'm fine leaving it in just to be explicit, thoughts?

Reply:
yeah, files sourced by bash don't need the shebang, that's only needed when directly executing a file. I would recommend not including the shebang as these aren't files that are intended to be directly executed.